recipes/cpp-smtpclient-library/all/conandata.yml

```diff
@@ -1,4 +1,7 @@
 sources:
+  "1.1.11":
+    url: "https://github.com/jeremydumais/CPP-SMTPClient-library/archive/refs/tags/v1.1.11.zip"
+    sha256: "d090ebcae75534a6f816174310fec046a96d18f4213581a4c5e75f7b173d624b"
   "1.1.10":
     url: "https://github.com/jeremydumais/CPP-SMTPClient-library/archive/refs/tags/v1.1.10.zip"
     sha256: "760e34417fa00b7bd56618f0d9e382867b65729c16175da1abb77d70fe1d1cd6"
```

Comment:
Let's remove the previous patch (`1.1.10`) and keep only the latest one (`1.1.11`)